Makefile

```diff
@@ -32,6 +32,7 @@ clean:
 	rm -rf python/az/aro/{aro.egg-info,build,dist} aro
 	find python -type f -name '*.pyc' -delete
 	find python -type d -name __pycache__ -delete
+	find -type d -name 'gomock_reflect_[1-9]*' -exec rm -rf {} \; 2>/dev/null
```

Comment:
fwiw: https://git.savannah.gnu.org/cgit/findutils.git/tree/find/exec.c?h=v4.8.0

Reply:
with all that said, did the pattern used in the two `find ... -delete`  above not work?
```
        find -type d -name 'gomock_reflect_[1-9]*' -delete
```
Assuming if there was a non-empty directory the above command would fail whereas `rm -rf` would succeed.

Reply:
@bryanro92 TIL! thanks for sharing that. Regarding the use of `find ... -delete`, the directory is not empty. All these folders have a `prog.go`. I could've deleted that file specifically before cleaning the folder but i went for this one step action